src/main/java/com/gotocompany/firehose/sink/common/blobstorage/oss/ObjectStorageService.java

```diff
@@ -0,0 +1,55 @@
+package com.gotocompany.firehose.sink.common.blobstorage.oss;
+
+import com.aliyun.oss.OSS;
+import com.aliyun.oss.OSSClientBuilder;
+import com.aliyun.oss.OSSException;
+import com.aliyun.oss.model.ObjectMetadata;
+import com.gotocompany.firehose.config.OSSConfig;
+import com.gotocompany.firehose.sink.common.blobstorage.BlobStorage;
+import com.gotocompany.firehose.sink.common.blobstorage.BlobStorageException;
+
+import java.io.ByteArrayInputStream;
+import java.io.File;
+import java.nio.file.Paths;
+
+public class ObjectStorageService implements BlobStorage {
+    private final OSS ossClient;
+    private final String bucketName;
```

Comment:
Should we set the OSSConfig as property? This approach encapsulates all related configuration settings within a single object, making the code cleaner and easier to maintain.

Reply:
i would also recommend this